websites/R/Reaper Scans/presence.ts

```diff
@@ -3,87 +3,110 @@ const presence = new Presence({
 	}),
 	elapsed = Math.floor(Date.now() / 1e3);
 
+enum Assets {
+	Logo = "https://i.imgur.com/tPMtbjL.png",
+	Search = "https://i.imgur.com/08wjeL0.png",
+	Read = "https://i.imgur.com/Gglu4Tk.png",
+	View = "https://i.imgur.com/BMzYEcO.png",
```

Comment:
Update the references too 
```suggestion
	Reading = "https://i.imgur.com/Gglu4Tk.png",
	Viewing = "https://i.imgur.com/BMzYEcO.png",
```